<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MsgDataFormatStringEditorDialog</class>
 <widget class="QDialog" name="MsgDataFormatStringEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Msg Data Format String Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="formatStringLabel">
     <property name="text">
      <string>Format String</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="formatStringLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="formatStringEditorLabel">
     <property name="text">
      <string>Format String Editor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="formatStringTextEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="textFormattingLabel">
     <property name="text">
      <string>Text Formatting</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addNewLineBtn">
       <property name="text">
        <string>Add New Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addTabBtn">
       <property name="text">
        <string>Add Tab</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="fieldLabel">
     <property name="text">
      <string>Field</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="fieldComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="addFieldBtn">
       <property name="text">
        <string>Add Field</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="operationlabel">
     <property name="text">
      <string>Operation</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="operandFirstComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="operationComboBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="operandSecondComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="addOperationBtn">
       <property name="text">
        <string>Add Operation</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Number Conversion</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="numConversionComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="addNumConversionBtn">
       <property name="text">
        <string>Add Conversion</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MsgDataFormatStringEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MsgDataFormatStringEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
